api/app/serializers/v1/reading_group_serializer.rb

```diff
@@ -5,26 +5,25 @@ class ReadingGroupSerializer < ManifoldSerializer
 
     abilities
 
-    typed_attribute :name, NilClass
-    typed_attribute :privacy, NilClass
-    typed_attribute :invitation_code, NilClass
-    typed_attribute :notify_on_join, NilClass
-    typed_attribute :memberships_count, NilClass
-    typed_attribute :annotations_count, NilClass
-    typed_attribute :highlights_count, NilClass
-    typed_attribute :created_at, NilClass
-    typed_attribute :current_user_role, NilClass
```

Comment:
I removed this line when I added types to the serializer. It is my impression that because the attribute is defined twice in the file, rails will only use the the last occurrence of the definition in the file (so this line is redundant). Is that correct?

Reply:
That is correct.